crates/cxx-qt-gen/src/syntax/lifetimes.rs

```diff
@@ -148,5 +150,7 @@ mod tests {
         assert_unsupported_type! { &dyn Foo };
         assert_unsupported_type! { fn(A) };
         assert_unsupported_type! { fn(i64) -> i32 };
+        // assert_unsupported_type! { Fn(i64, i64) -> i32 }; // This currently just panics so test fails
```

Comment:
we're trying to hit `PathArguments::Parenthesized` so maybe needs to be like  `assert_unsupported_type! { Vec<Fn(i64, i64) -> i32> }` as it should be in the arguments of a path?

Reply:
Hmm no i don't think that is quite right either, seems it the path part would be the `Fn`

Reply:
Lets dig into it later :sweat_smile: